Marlin/src/module/temperature.h

```diff
@@ -559,6 +559,10 @@ class Temperature {
       static int read_max6675();
     #endif
 
+    #if ENABLED(HEATER_1_USES_MAX6675)
+      static int read_max6675_2();
```

Comment:
Any way that a single `read_max6675` function can be augmented to handle more than 1 thermocouple?